.github/workflows/bors.yml

```
@@ -75,21 +75,9 @@ jobs:
 
       - uses: actions/checkout@v4
 
-      - name: update Mathlib.lean
-        run: |
-          git ls-files 'Mathlib/*.lean' | LC_ALL=C sort | sed 's/\.lean//;s,/,.,g;s/^/import /' > Mathlib.lean
-
-      - name: update Mathlib/Tactic.lean
-        run: |
-          git ls-files 'Mathlib/Tactic/*.lean' | LC_ALL=C sort | sed 's/\.lean//;s,/,.,g;s/^/import /' > Mathlib/Tactic.lean
-
-      - name: update Counterexamples.lean
-        run: |
-          git ls-files 'Counterexamples/*.lean' | LC_ALL=C sort | sed 's/\.lean//;s,/,.,g;s/^/import /' > Counterexamples.lean
-
-      - name: update Archive.lean
+      - name: import files in {Mathlib Mathlib/Tactic Counterexamples Archive}.lean
```

How about one of these?
```suggestion
      - name: update import-only files {Mathlib Mathlib/Tactic Counterexamples Archive}.lean
 ```
```suggestion
      - name: update lists of files in {Mathlib Mathlib/Tactic Counterexamples Archive}.lean

---

I went for the first option: `update import-only files`